src/olympia/editors/templates/editors/queue.html

```diff
@@ -162,13 +162,7 @@
             {% for value in row %}
               <td>{{ value|xssafe }}</td>
             {% endfor %}
-            {% if table.show_version_notes %}
-            <td>
-              <div class="addon-version-notes">
-                <a href="#" class="ed-sprite-notes app-icon" title="{{ _('Version Notes / Notes for Reviewer') }}">&nbsp;</a>
-              </div>
-            </td>
-            {% endif %}
+            <td>&nbsp;</td>
```

Comment:
Not sure I follow this - if we don't a version_notes column in any of the queues why not just adjust the css?